tiledb/api/src/metadata.rs

```diff
@@ -20,9 +20,6 @@ pub enum Value {
     Int64Value(Vec<i64>),
     Float32Value(Vec<f32>),
     Float64Value(Vec<f64>),
-    //StringAsciiValue(CString),
```

Comment:
Is there no support for strings or is the idea to return then as a Vec<u8> and rely on external bits to do the conversion?

Reply:
That seems to be how we've generally done things, string goes in as bytes and the user is responsible for checking UTF-8 when it comes out.

Since core doesn't validate UTF-8 I'm not sure we really have another option.  We could add a `From<String>` or something though so we can move the contents of a string into metadata